docs/appendices/migration-guides/index.mdx

```diff
@@ -106,6 +106,32 @@ it. The main use case for this variable was to change `npm run lint` behavior so
 that it triggers a warning instead of an error when a deprecated GraphQL is
 used. This is now the default behavior.
 
+### New Adobe Commerce B2B PHP module
+
+Some B2B features were previously leveraging features from the
+[Magento2 Commerce module](/docs/magento2/commerce#magento2-commerce-module-installation).
+During this release, we've extracted B2B related features into their own
+repository.
+
+**If you work on an Adobe Commerce B2B project**, you must:
+
+- update `front-commerce/magento2-commerce-module` to
+  [version 2.x](https://gitlab.com/front-commerce/magento2-commerce-module-front-commerce/-/releases)
+  (currently `2.0.0`)
```

Comment:
is there any other breaking change in the 2.x version ? In any case, it might be interesting to add some details about that as well.

Reply:
None.
The major version bump was exclusively related to this change.